### ISSUE TITLE
If-Modified-Since = 0 raise java.lang.IllegalArgumentException: Invalid format: "0" exception

### ISSUE THREAD

**Issue by AlexanderRay** [lynndylanhurley/ng-token-auth#126]
``` coffee 
    angular.forEach(httpMethods, (method) ->
      $httpProvider.defaults.headers[method] ?= {}
      $httpProvider.defaults.headers[method]['If-Modified-Since'] = '0'
    )
```

raise a runtime exception in play framework 2.3.8.

```
java.lang.IllegalArgumentException: Invalid format: "0"
        at org.joda.time.format.DateTimeFormatter.parseDateTime(DateTimeFormatter.java:873) ~[joda-time-2.3.jar:2.3]
        at controllers.AssetInfo$.parseDate(Assets.scala:106) ~[play_2.11-2.3.7.jar:2.3.7]
        at controllers.AssetsBuilder$$anonfun$controllers$AssetsBuilder$$maybeNotModified$3.apply(Assets.scala:326) [play_2.11-2.3.7.jar:2.3.7]
        at controllers.AssetsBuilder$$anonfun$controllers$AssetsBuilder$$maybeNotModified$3.apply(Assets.scala:325) [play_2.11-2.3.7.jar:2.3.7]
        at scala.Option.flatMap(Option.scala:171) [scala-library-2.11.5.jar:na]
```

can you change it to

``` coffee 
    angular.forEach(httpMethods, (method) ->
      $httpProvider.defaults.headers[method] ?= {}
      $httpProvider.defaults.headers[method]['If-Modified-Since'] = 'Mon, 26 Jul 1997 05:00:00 GMT'
    )
```

**Comment by lynndylanhurley** [comment]
Sure thing, thx for the report.

**Comment by lynndylanhurley** [comment]
Just fixed in version `0.0.25-beta1` - please confirm!
